src/registrar/fixtures.py

```diff
@@ -123,6 +128,12 @@ class UserFixture:
             "last_name": "DiSarli-Analyst",
             "email": "gaby@truss.works",
         },
+        {
+            "username": "cfe7c2fc-e24a-480e-8b78-28645a1459b3",
+            "first_name": "Nicolle",
+            "last_name": "LeClair",
```

Comment:
```suggestion
            "first_name": "Nicolle-Analyst",
            "last_name": "LeClair-Analyst",
```
Because of how users are displayed, if your staff user has the same name as your Admin user, it will basically look like there is a duplicate record (though there isn't). For clarity, we add '-Analyst' at the end of first/last names for staff users so we can easily differentiate them when doing fixtures, testing, etc.